data-pipeline.yml

```diff
@@ -48,6 +49,7 @@ tasks:
     routes:
       - notify.email.release-mgmt-analysis@mozilla.com.on-failed"
       - notify.irc-channel.#bugbug.on-failed
+      - index.project.relman.bugbug.bugbug_data.latest
```

Comment:
Two tasks can't have the same index path, or the one finishing last will overwrite the first.

We should update the trainer script to use two different URLs.